docs/en/Contributing-Getting-Started.md

```diff
@@ -0,0 +1,261 @@
+---
+layout: global
+title: Contributor Getting Started
+nickname: Contributor Getting Started
+group: Resources
+---
+
+Welcome to the Alluxio community! We are excited for your contributions and engagement in the
+community! Here is a guide on how to get started with becoming a member in the Alluxio open source
+community.
+
+## Requirements
+
+The main requirement is a computer with Mac OSX or Linux installed. Alluxio does not have Windows support at the moment.
+
+## Software and Account Preparation
+
+There are several accounts and setup required before contributing to Alluxio.
+
+### Java 7 or 8
+
+Developing Alluxio requires Java 7 or 8. If you are unsure of which version of java you have, you
+can check with this command:
+
+```bash
```

Comment:
Ack. Seems like other docs have code sections in `_include/` to better support different languages. I'm fine with this for now because this is a very most important starting page. Later on when new languages are added, we can do such refactoring.